frontend/src/components/activePage/ActiveTemplates/ActiveTemplates.stories.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/no-multi-comp */
+import ActiveTemplates, { IActiveTemplatesProps } from './ActiveTemplates';
+import { Story } from '@storybook/react';
+import { Title, Description, Stories } from '@storybook/addon-docs/blocks';
+import { someKeysOf } from '../../../utils';
+import { courses, templates } from '../utils/data';
+
+export default {
+  title: 'Components/ActivePage/ActiveTemplates',
+  component: ActiveTemplates,
+  argTypes: { destroyAll: { action: 'clicked' } },
+  parameters: {
+    docs: {
```

Comment:
It is great you added the docs to the storry!

If you want and when you have time, you could make another PR where you add instructions in the guidelines on how to do it. The docs on the stories would still be optional, but if someone would like to add them, he/she would already know how to do it